.circleci/config.yml

```diff
@@ -158,7 +158,12 @@ jobs:
           name: Install dependencies
           command: |
             apt-get install -y wget jq make gettext gzip
+            apt install -y python3.11-venv
```

Comment:
Does it have to be 3.11 because the earlier version is not available?

Reply:
It's the version of python pre-installed in the image we're using. It might be possible to make the installation less specific (`install python3-venv` instead), will investigate.